src/SaveApis.Common/Application/Hangfire/HangfireQueue.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SaveApis.Common.Application.Hangfire;
+
+public enum HangfireQueue
+{
+    Event,
```

Comment:
Value wird nur für RecurringEvents verwendet.
```suggestion
    RecurringEvent,
```